### PR TITLE
merge HUMAN_MAX_CREDITS and ALIEN_MAX_CREDITS since they're the same

### PR DESCRIPTION
**Pull request by bmorel**

### CODE REVIEW

File: src/shared/bg_gameplay.h [hunk 1]
```diff
@@ -35,6 +35,7 @@ along with Unvanquished Source Code.  If not, see <http://www.gnu.org/licenses/>
 
 #define BUILDER_DECONSTRUCT_RANGE 100
 
+constexpr int MAX_CREDITS = 2000;
```
Comment (necessarily-equal) [inline src/shared/bg_gameplay.h:38]: How about keeping the comment? You are the first one to complain about lack of doc :P
```suggestion
constexpr int MAX_CREDITS = 2000; // for aliens, CREDITS_PER_EVO converts this to evos for display
```

**Review by necessarily-equal — APPROVED** [review]
A nitpick, otherwise, LGTM

**Comment by Gireen** [comment]
I don't think this values should be merged.
Its brings no benefit but makes it more work if we later want to have different values per team again.

Also merging a PR in less than 24 hours is not ok imo.

**Comment by bmorel** [comment]
I'll revert then.

**Comment by necessarily-equal** [comment]
If you want, I can revert it except making it TEAM_* agnostic.

**Pull request merged**